gdk/src/lib.rs

```diff
@@ -100,6 +100,7 @@ pub use crate::atom::TARGET_PIXMAP;
 pub use crate::atom::TARGET_STRING;
 pub use crate::change_data::ChangeData;
 pub use crate::event::Event;
+pub use crate::event::FromEvent;
```

Comment:
```suggestion
pub use crate::event::{Event, FromEvent};
```

Reply:
I was following the `atom` exports example from above, but OK.